frontend/src/app/components/FeaturedExperiment/index.js

```diff
@@ -45,7 +45,8 @@ export default class FeaturedExperiment extends React.Component {
   render() {
     const { experiment, enabled, installed, hasAddon, sendToGA,
       eventCategory } = this.props;
-    const { description, title, subtitle, slug, video_url, error } = experiment;
+    const { description, title, subtitle, slug, video_url, error, platforms } = experiment;
+    const isMobile = platforms.includes("ios") || platforms.includes("android");
```

Comment:
nit: we use `isMobile` elsewhere to determine whether or not the user is on a mobile device based on useragent. I would slightly rename this to not cause confusion.

Reply:
cool